### DISCUSSION TITLE
Timeout api.osv.dev and retry with osv-scanner

### DISCUSSION THREAD

**Discussion by lfandrade** [google/osv-scanner#916] [Q&A] [answered]
Hello everybody,

I'am implements the osv-scanner into a jenkins pipeline that run a process into macmini with paramiko. So i realized many calls to osv-scanner -lockfile=pubspec.lock , my file have 500 packages and sometimes a message bellow is showing. I've search in github osv.dev and osv-scanner and i see that exist a issue about timeout .

I need help then how to implements a retry when a timeout is returned by osv-scanner

Message osv-scanner -> API query failed: osv.dev query failed: Post "https://api.osv.dev/v1/querybatch": dial tcp 142.251.129.211:443: i/o timeout

**Comment by oliverchang** [comment, marked as answer]
Hi, thanks for creating this. 

Which version of OSV-Scanner are you using? 

I believe @another-rex added retries recently to OSV-Scanner in 1.7.1 https://github.com/google/osv-scanner/releases/tag/untagged-ee8b7e50bafb28ef9b04. 

Are you able to check if using this resolves your issue?
